shell/components/ResourceList/index.vue

```diff
@@ -124,7 +128,7 @@ export default {
         return [];
       }
 
-      return this.$store.getters['type-map/headersFor'](this.schema);
+      return this.$store.getters['type-map/headersFor'](this.schema, this.canPaginate);
```

Comment:
By any chance will pagination be a part of the type schema? If so this probably isn't necessary.

Reply:
The BE will support pagination of all steve types, though this kind of stuff is because we live in a halfway world. Ultimately almost all resources will be paginated. If we ever get to the stage where all are we can rip this kind of thing out.